Makefile

```diff
@@ -16,55 +16,22 @@ TEST_SCRIPT_NAME = run_tests.sh
 
 default: compile
 
-all: compile build build-startup-script build-executable-script build-test-script
+all: compile build set-executable
 
 compile:
 	kb-sdk compile $(SPEC_FILE) \
 		--out $(LIB_DIR) \
 		--pysrvname $(SERVICE_CAPS).$(SERVICE_CAPS)Server \
 		--pyimplname $(SERVICE_CAPS).$(SERVICE_CAPS)Impl;
 
-build:
+set-executable:
```

Comment:
remove all the unused script generation stuff and replace it with a shortcut to set the executable bit on the scripts (just in case it somehow gets unset).